Mobile Buy SDK/Mobile Buy SDK/Data/BUYClient+Storefront.m

```diff
@@ -173,6 +173,27 @@ - (NSOperation *)getCollectionsPage:(NSUInteger)page completion:(BUYDataCollecti
 	}];
 }
 
+- (NSOperation *)getCollectionsByIds:(NSArray<NSString *> *)collectionIds completion:(BUYDataCollectionsBlock)block
```

Comment:
We will need to update the Android SDK, it is actually ignoring the page and only doing getCollectionsByIds(List<> collectionIds)